sql/sql_class.h

```diff
@@ -3323,6 +3323,13 @@ class THD : public MDL_context_owner,
   Gtid_set owned_gtid_set;
 #endif
 
+#ifdef HAVE_OPENSSL
+  std::string m_connection_certificate;
+#endif
+
+  std::string const &connection_certificate() const;
```

Comment:
`noexcept`

Reply:
Technically, he std::string copy constructor could throw a bad_alloc.

Reply:
No, no copy constructor here, the result is returned by const reference
`std::string const &`

Reply:
Uh, right, I completely missed the & aligned to the function name, fixed.